### ISSUE TITLE
Mod 3, exercise 1, upload templates - not able to upload correctly.

### ISSUE THREAD

**Issue by chaast** [MicrosoftLearning/MB-300-Microsoft-Dynamics-365-Core-Finance-and-Operations#43]
# Module: 3
#### Step: 6

Description of issue

not all the entities exist in the target environment - which becomes apparent when uploading the templates in step 6. please see attached image for the issue.

![image](https://user-images.githubusercontent.com/89986870/165059295-4bc922ca-9a79-4c62-a2c4-fc33327f9d55.png)

Repro steps:

1. all the other exercises in the lab were followed and completed to the letter. 

during the course we did experience latency issues from the virtual machine - potentially due to not having sufficient storage capacity in the VM... could this have been the cause of the problem? If so, what is the recommended / minimum set up requirement of the machines please?

If this issue is due to something else, please advise the required steps.

thank you.

**Comment by hakimmartin** [comment]
Hi @chaast - The performance of the lab VM on the lab hoster’s (e.g Skillable) lab environment provided to the student for execution of the instructions. Performance issue may be due to low internet bandwidth at the student’s end or generally low performance on the hoster’s end. Usually, during the live class the instructor can help check the issues with hoster by raising a support ticket with the hoster they are using.

**Comment by chaast** [comment]
@hakimmartin 

to clarify - this issue i faced is definitely due to performance issues with internet bandwith? And not due to a problem with the lab???  can you confirm this please

**Comment by TesterTesterson1004** [comment]
**Can we please reopen this issue for a deeper discussion & analysis?**  I experience the same issue, at the same place.  My instance has not experienced any performance or other issues/errors within the VM at my ALH - this doesn't feel like an issue related to poor performance to me, but I cannot say I am experienced with this application.

In Step 5-6 of this Exercise, where we load all default templates, this completes and populates w/templates - but there are a series of alerts after the template load completes.  For example:
 
![image](https://user-images.githubusercontent.com/49700217/167977818-0ad002ec-659c-4b3a-b9b4-31fd582b1274.png)

The message details for that look like so:

![image](https://user-images.githubusercontent.com/49700217/167977900-78e113c3-bfbd-42d3-963c-f64085ebe4c6.png)

After loading templates, I select '_020 - GL Shared_', and the instructions tell us to '_View available entities_' for this shared template.  Then, step 9 says to "_...In the list, find and select the desired record, such as Main account categories by using the filter..._"

However, as can be seen in this screenshot, there is only a single entity present - '_Number Sequence Group_' - and no hint of any other entities, much less one called '_Main account categories_':

![image](https://user-images.githubusercontent.com/49700217/167977076-7b4db515-8461-4357-83e5-10395c0b54bc.png)

From the warning/errors on template load, it looks like the 'environment' is missing something....but what?  And how to load and/or configure whatever it is that is preventing this lab instructions from being able to be followed and progress further?

**Comment by TesterTesterson1004** [comment]
I have very little understanding of the F&O VM this is built on, (probably very few people do - except for those at Microsoft that wrote it) however I played with the problem a bit more - started a few fresh instances to try and pin down that the error referenced reproed consistently every time, and it did.

Next, I tried an 'entity refresh' method I saw referenced elsewhere:  After loading the templates in fresh instance, repro'ing the error, I then did the following:

- Go to '_Data management_', '_Framework parameters_', '_Entity settings_', and selected '_Refresh entity list_'

- Wait from 5-7 minutes, and a notification appeared:

![image](https://user-images.githubusercontent.com/49700217/167987386-57699a3a-2669-4ad3-b134-ae58fcd9df78.png)

- After I see this Action Center notification, went back to the troublesome exercise and executed the '_Load default templates_' & '_Load all_' again and this time....I now have entities!

![image](https://user-images.githubusercontent.com/49700217/167987539-ce8c6cef-5d53-4428-a92b-a63706eeded1.png)

Now, who can explain what this means, and if it is actually the correct thing to do to fix for this issue?
